server/src/main/java/io/druid/query/lookup/LookupModule.java

```diff
@@ -146,24 +174,15 @@ public Object get(String id)
           }
 
           @Override
-          public Map<String, LookupExtractorFactory> getAll()
+          public LookupsState getAll()
```

Comment:
this is better as it gives the accurate state of the node. I'm actually planning to remove old apis and have only newer apis. older POST api is kept only to keep things backwards compatible. is there any reason to keep them around?